app/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDao.java

```diff
@@ -89,6 +89,7 @@ public UserConsent getActiveUserConsent(String healthCode, SubpopulationGuid sub
 
         DynamoDBQueryExpression<DynamoUserConsent3> query = new DynamoDBQueryExpression<DynamoUserConsent3>()
             .withScanIndexForward(false)
+            .withLimit(1)
             .withQueryFilterEntry("withdrewOn", new Condition().withComparisonOperator(ComparisonOperator.NULL))
             .withHashKeyValues(hashKey);
```

Comment:
If it *is* possible to withdraw from the second consent but not the first, can we have a test for that to make sure this query works as expected?